### DISCUSSION TITLE
IRAF on new 2021 iMacs

### DISCUSSION THREAD

**Discussion by kkwitter** [iraf-community/iraf#155] [Q&A] [answered]
What is the situation for IRAF on the new iMacs?

**Comment by olebole** [comment, marked as answer]
There is ongoing work to support the IRAF community distribution for the new CPU architecture directly: #131. This is however currently stalled, as I don't have access to a M1 Mac, so I can't test it. If you can temporarily provide me ssh access, I can finish this.
Using a 64-bit version precompiled on a x86_64 CPU was reported to work.
[x11iraf](https://iraf-community.github.io/x11iraf) should compile and work, but is still untested. The required XQuartz dependency [claims to have M1 support](https://www.xquartz.org/releases/XQuartz-2.8.0.html).

**Comment by kkwitter** [comment]
I don’t have an M1 yet, either! Let’s hope you are able to test it at some point. Thanks for the quick reply.

-----------------------------
Karen B. Kwitter
E. Fitch Professor Emerita
Astronomy Department
Williams College
Williamstown, MA 01267 USA
------------------------------
